clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -149,7 +149,11 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
 
     def tags = GCEUtil.buildTagsFromList(description.tags)
 
-    def serviceAccount = GCEUtil.buildServiceAccount(description.authScopes)
+    if (description.authScopes && !description.serviceAccountEmail) {
+      description.serviceAccountEmail = "default"
```

Comment:
Does the email have to be well-formed? If I had to guess it will complain because it doesn't match some regex.